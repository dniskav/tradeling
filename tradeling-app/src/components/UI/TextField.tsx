import React from 'react';
import styled from 'styled-components';

const InputText = styled.input`
    border: 1px solid #999;
    padding: 14px 16px;
    box-sizing: border-box;
    width: 360px;
    font-size: 16px;
    letter-spacing: 0.8px;
`;

interface ITextField {
  placeholder: string;
}

const TextField: React.FC<ITextField> = ({ placeholder }) => {

  return (
    <InputText type="text" placeholder={placeholder} />
  )
}

export default TextField;
