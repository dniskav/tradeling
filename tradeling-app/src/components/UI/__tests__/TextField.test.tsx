import { shallow } from "enzyme";
import TextField from '../TextField';

jest.mock("react-redux", () => {
  return {
    useSelector: () => [],
    useDispatch: () => jest.fn(),
  }
});

describe("<TextField />", () => {
  const tree = shallow(<TextField />);
  it("TextField should match snapshot", () => {
    expect(tree).toMatchSnapshot();
  });
});
