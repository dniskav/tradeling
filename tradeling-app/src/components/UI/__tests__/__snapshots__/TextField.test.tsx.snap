// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField /> TextField should match snapshot 1`] = `<styled.input />`;
